packages/astro/package.json

```diff
@@ -32,7 +35,7 @@
     "./jsx/*": "./dist/jsx/*",
     "./jsx-runtime": "./dist/jsx-runtime/index.js",
     "./config": "./config.mjs",
-    "./internal": "./internal.js",
+    "./compat": "./dist/compat/index.js",
```

Comment:
`compat` feels so much like `preact/compat`. Can `./config` just export this instead? One less concept for a user to learn, one less entrypoint to document, etc.

Reply:
Good point. I can put it in `config` but I didn't think we'd want to pull in all these deps (vite, etc) when loading the config (which exports `defineConfig`)